extension-manifest-v3/src/store/options.js

```diff
@@ -162,13 +155,19 @@ export function observe(property, fn) {
       const prevValue = value;
       value = options[property];
 
-      fn(value, prevValue);
+      try {
+        fn(value, prevValue);
+      } catch (e) {
+        console.error(`Error while observing options: `, e);
+      }
     }
   };
 
   observers.add(wrapper);
 
-  store.resolve(Options);
```

Comment:
Add await and async in the function definition.

Reply:
`observe` is called in top level most of the time where it cannot be awaited.

this is also why the we trigger callback (wrapper) after the `store.resolve` - so that observers are always notified, regardless on when the `observe` was called.

Reply:
but I see the point with the adblocker startup. make sense